rep-2004.rst

```diff
@@ -0,0 +1,676 @@
+REP: 2004
+Title: Package Quality Categories
+Author: William Woodall <william@openrobotics.org>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 17-Dec-2019
+Post-History: 17-Dec-2019
+
+
+Abstract
+========
+
+This REP describes a set of categories meant to convey the quality or maturity of packages in the ROS ecosystem.
+Inclusion in a category, or quality level, is based on the policies to which a package adheres.
+The categories address version, change control, documentation, testing, dependency and platform support policies.
+
+Motivation
+==========
+
+The purpose of this REP is to provide standard guidelines regarding package quality, for both maintainers and consumers.
+
+The categories are meant to set quality expectations for package consumers, not to enforce quality rules on maintainers.
+Package maintainers are responsible for establishing processes and documenting how their package's policies achieve a certain quality level.
+With the help of documented policies, package consumers can better evaluate whether a package or its dependencies meet the standards for use in their projects.
+
+By providing these rough goals for package maintainers to strive towards, the categories will also encourage better quality across the ROS ecosystem.
+
+Specification
+=============
+
+There are four quality levels below, each roughly summarized as:
```

Comment:
I find it odd that you start out saying there's 4 levels, but below you specify 5.

Reply:
The fifth ("default") level was added afterwards, so it's probably just a typo.